main.tf

```diff
@@ -40,5 +40,52 @@ resource "azurerm_dns_a_record" "wowza" {
   resource_group_name = var.dns_resource_group
   ttl                 = 300
   records             = [var.lb_IPaddress]
-  tags                = local.common_tags
-}
\ No newline at end of file
+
+  tags = local.common_tags
+}
+
+# =================================================================
+# =================    automation account    ======================
+# =================================================================
+resource "azurerm_automation_account" "vm-start-stop" {
```

Comment:
Can this be moved to its own file called 'automation.tf' with the wowza module files please?

Reply:
Sure. 
The reason for keeping it outside was that it would make adding multiple runbooks to the same AA easier. However, I can do this with a for each loop.

Reply:
moved to automation.tf file within wowza module